include/wlr/types/wlr_surface.h

```diff
@@ -36,6 +36,11 @@ struct wlr_surface_state {
 	struct wl_listener buffer_destroy;
 };
 
+struct wlr_surface_role {
+	const char *name;
```

Comment:
Note that this is only here for debugging purposes, and isn't really useful anymore.